src/main/proto/ga4gh/reads.proto

```diff
@@ -31,37 +31,40 @@ message ReadGroup {
 
   // The ID of the dataset this read group belongs to.
   string dataset_id = 2;
+  
+  // The ID of the read group set this read group belongs to.
+  string read_group_set_id = 3;
 
   // The read group name.
-  string name = 3;
+  string name = 4;
```

Comment:
Yes, thanks! I've seen the future compatibility benefits of fixing field numbers and I'm happy to work with whatever pattern. I understand the reasoning for maintaining field numbers across released versions. Perhaps we can agree on a future release where the field numbers will be fixed?

In practice, do you pick large field numbers for fields that you might place towards the end of a message? For perhaps pedantic reasons I would like to place the `info` fields last in the order of tags. Does it become unavoidable to place tag numbers out of order from their vertical position in the .proto?